torch/fx/experimental/proxy_tensor.py

```diff
@@ -433,13 +436,30 @@ def create_arg(self, a: Any):
         return super().create_arg(a)
 
 
+DISPATCH_TRACE_ERROR_PRINT_GRAPH = False
+
 def dispatch_trace(
         root: Union[torch.nn.Module, Callable],
         tracer: Tracer,
         concrete_args: Optional[Tuple[Any, ...]] = None,
 ) -> GraphModule:
-    graph = tracer.trace(root, concrete_args)
     name = root.__class__.__name__ if isinstance(root, torch.nn.Module) else root.__name__
+    try:
+        graph = tracer.trace(root, concrete_args)
+    except Exception:
+        if DISPATCH_TRACE_ERROR_PRINT_GRAPH:
+            gm_printed = GraphModule(tracer.root, tracer.graph, name).print_readable(print_output=False)
+            log.info(f"""
+            DISPATCH_TRACE_ERROR_PRINT_GRAPH: \n
+            {gm_printed} \n
+            """)
+        else:
+            log.warning("""
+            Dispatch trace exception,
+            set torch.fx.experimental.proxy_tensor.DISPATCH_TRACE_ERROR_PRINT_GRAPH = True
+            to print the incomplete graph.
+            """)
```

Comment:
The indentation on the string is going to look super ugly when printed. Please dedent it to the left.

Reply:
If you really care about maintaining the relative indent, use string juxtaposition
```
log.warning(
    "Foo "
    "bar "
    "baz."
)
```